consensus/hotstuff/verification/combined_test.go

```diff
@@ -119,8 +119,9 @@ func TestCombinedQC(t *testing.T) {
 	assert.True(t, valid, "original QC should be valid")
 
 	// verification with missing identity should be invalid
-	_, err = signers[0].VerifyQC(voterIDs[:minShares-1], qc.SigData, block)
-	assert.Error(t, err, "verification of QC should fail with missing voter ID")
+	valid, err = signers[0].VerifyQC(voterIDs[:minShares-1], qc.SigData, block)
+	require.NoError(t, err)
+	assert.False(t, valid, "verification of QC should fail with missing voter ID")
 
```

Comment:
adding this comment here as I can't add a comment to the end of the file:

It would be great to test that `VerifyQC` actually returns a `model.ErrInvalidSigner` if the ID of one of the signers is unknown / invalid. Would suggest to add the following to the _end_ of the `TestCombinedQC` test method:


```golang
	// verification of a QC with an unknown voter ID should fail
	withUnknownVoter := append(voterIDs, unittest.IdentifierFixture())
	valid, err = signers[0].VerifyQC(withUnknownVoter, qc.SigData, block)
	require.True(t, errors.Is(err, model.ErrInvalidSigner))
	assert.False(t, valid, "QC with an unknown voter ID should be invalid")
```